README.md

```
@@ -24,11 +33,12 @@ Want to build trading bots without code? Check out our new platform [BotSpot](ht
 
 **Visit [BotSpot.trade](https://botspot.trade/sales?utm_source=lumibot+docs&utm_medium=documentation&utm_campaign=GitHub+Readme) to get started building AI-powered trading bots today!**
 
-## Blog
+## Learn More
 
-Our blog has lots of example strategies and shows you how to run a bot using LumiBot. Check the blog out here:
+Check out example strategies and tutorials on our blog, or use our AI agent to build strategies for you:
 
-**https://lumiwealth.com/blog/**
+**Blog:** https://lumiwealth.com/blog/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)